test/common/http/conn_manager_utility_test.cc

```diff
@@ -1018,5 +1019,40 @@ TEST_F(ConnectionManagerUtilityTest, RemovesProxyResponseHeaders) {
   EXPECT_FALSE(response_headers.has("proxy-connection"));
 }
 
+// Verify when appending x-forwarded-port is turned on, the mutated header should have
```

Comment:
Can you extend a test to ensure that when appendXForwardedPort is false nothing gets set?

Reply:
Sure, done in 0f30480.